tests/consumers/acero_consumer.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pyarrow as pa
+import pyarrow.substrait as substrait
+
+
+class AceroConsumer:
+    """
+    Implementation of the acero substrait consumer Class for testing
```

Comment:
```suggestion
    Implementation of the acero substrait consumer for testing
```

Or maybe, "adapts the Acero Substrait consumer to the test framework"?